third_party/ibis/ibis_cloud_spanner/to_pandas.py

```diff
@@ -34,9 +37,17 @@ def to_pandas(snapshot, sql, query_parameters):
         for row in data_qry:
             data.append(row)
 
-        columns = [f.name for f in data_qry.fields]
+        columns = [_.name for _ in data_qry.fields]
+        bytes_columns = [
+            _.name for _ in data_qry.fields if _.type_.code == TypeCode.BYTES
+        ]
```

Comment:
```suggestion
        columns = [field.name for field in data_qry.fields]
        bytes_columns = [
            f.name for f in data_qry.fields if f.type_.code == TypeCode.BYTES
        ]
```
I'd say, when possible, we could keep our code simple, easy to read and maintain. That's the case here for me, but since it doesn't change the result per se, feel free to skip this suggestion if you feel like!

Reply:
Thanks for the review Helen.

I use underscore variables for variables that are temporary in nature and can be ignored. i.e. I have no intention of using `field` or `f` anywhere else.

Personally I find the suggested code more confusing because we use `field` to loop through `fields` the first time and `f` the second time.

I understand it is personal preference but I think I'll keep the underscore version for now - I'm sure the next person in this code will change it to match their preference :-D

Thanks again.